themes/ddbasic/template.node.php

```diff
@@ -366,3 +366,30 @@ function ddbasic_preprocess__node__ding_eresource(&$variables) {
       break;
   }
 }
+
+/**
+ * Ding Page.
+ */
+function ddbasic_preprocess__node__ding_page(&$variables) {
+  switch ($variables['view_mode']) {
+    case 'teaser':
+      if (!empty($variables['field_ding_page_list_image'][0]['uri'])) {
+        $variables['background_image'] = image_style_url('ding_list_square', $variables['field_ding_page_list_image'][0]['uri']);
+      } else {
+        $variables['background_image'] = '';
+      }
+      break;
+    case 'teaser_no_overlay':
+      array_push($variables['classes_array'], 'node-teaser-no-overlay');
+
+      if (!empty($variables['field_ding_page_list_image'][0]['uri'])) {
+        $variables['background_image'] = image_style_url('ding_list_square', $variables['field_ding_page_list_image'][0]['uri']);
+      } else {
```

Comment:
"else" should always start on new-line (See https://www.drupal.org/docs/develop/standards/coding-standards#controlstruct).